include/graphics.c

```diff
@@ -40,16 +46,123 @@ int windowInit() {
     window = SDL_CreateWindow("TicTacToe", SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, windowWidth, windowHeight, SDL_WINDOW_SHOWN);
 
     if (!window) {
-        printf("Couldn't create window! Reason: %s", SDL_GetError());
+        printf("Couldn't create window! Reason: %s\n", SDL_GetError());
         return 1;
     }
 
     windowSurface = SDL_GetWindowSurface(window);
+
+#if SDL_BYTEORDER == SDL_BIG_ENDIAN
+    Rmask = 0xFF000000;
+    Gmask = 0x00FF0000;
+    Bmask = 0x0000FF00;
+    Amask = 0x000000FF;
+#else
+    Rmask = 0x000000FF;
+    Gmask = 0x0000FF00;
+    Bmask = 0x00FF0000;
+    Amask = 0xFF000000;
+#endif
+
+    bgSurface = SDL_CreateRGBSurface(NULL, windowWidth, windowHeight, 32, Rmask, Gmask, Bmask, Amask);
+
+    DrawField();
+
     return 0;
 }
 
-void DrawField() {
-    SDL_FillRect(windowSurface, NULL, SDL_MapRGB(windowSurface->format, 0, 0, 0));
+void ResetScreen() {
+    SDL_BlitSurface(bgSurface, NULL, windowSurface, NULL);
     SDL_UpdateWindowSurface(window);
 }
 
+void InitStdText() {
+    smallFont = TTF_OpenFont(RESOURCEDIR"Serif.ttf", 30);
+    if (!smallFont) {
+        printf("Couldn't load small font! Reason: %s\n", SDL_GetError());
+        return;
+    }
+
+    bigFont = TTF_OpenFont(RESOURCEDIR"Serif.ttf", 58);
+    if (!bigFont) {
+        printf("Couldn't load big font! Reason: %s\n", SDL_GetError());
+        return;
+    }
+
+    ultraSmallFont = TTF_OpenFont(RESOURCEDIR"Serif.ttf", 25);
+    if (!bigFont) {
+        printf("Couldn't load ultra small font! Reason: %s\n", SDL_GetError());
+        return;
+    }
+
+    SDL_Color textColor = {255, 255, 255, 255};
+
+    SDL_Rect namePos = {windowWidth-170, 40, 60, 70};
+    SDL_Surface *name = TTF_RenderText_Blended(smallFont, "TicTacToe", textColor);
+    SDL_BlitSurface(name, NULL, bgSurface, &namePos);
+    SDL_FreeSurface(name);
+
+    SDL_Rect madeByPos = {windowWidth-170, 130, 70, 80};
+    SDL_Surface *madeBy = TTF_RenderText_Blended_Wrapped(ultraSmallFont, "Made by Teufelchen1 and kosmas12", textColor, 170);
+    SDL_BlitSurface(madeBy, NULL, bgSurface, &madeByPos);
+    SDL_FreeSurface(madeBy);
+
+    SDL_Rect copyrightPos = {20, 420, 100, 60};
+    SDL_Surface *copyright = TTF_RenderText_Blended(smallFont, "Copyright 2021 and licensed under the GPL3", textColor);
+    SDL_BlitSurface(copyright, NULL, bgSurface, &copyrightPos);
+    SDL_FreeSurface(copyright);
+
+}
+
+void UpdateScore() {
+    // Make text white
+    SDL_Color textColor = {255, 255, 255, 255};
+
+    SDL_Rect pScorePos = {10, 0, 100, 62};
+    int pScoreLen = 8;
```

Comment:
How about changing this to:
```C
#define pScoreLen 8
SDL_Rect pScorePos[pScoreLen] = {10, 0, 100, 62};
```
Ultimately, everything thats static should be doable via #defines I think.